utils/spacewalk-clone-by-date

```diff
@@ -106,7 +106,7 @@ def merge_config(options):
                  "security_only", "use_update_date", "no_errata_sync",
                  "errata", 'dry_run']
     for key in overwrite:
-        if config.has_key(key) and not getattr(options, key):
+        if key in config and not getattr(options, key):
```

Comment:
`config` is a dict: https://github.com/uyuni-project/uyuni/pull/458/commits/d7a39e2ceb44026ed028f57ecf2389edf9b6fb94#diff-12ad85f09c55ba2f92e47253e811ab68R98